system/jaeger-operator/values.yaml

```diff
@@ -0,0 +1,14 @@
+# Default values for jaeger.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+#
+global:
+  region: DEFINED_IN_VALUES_FILE
+
+jaeger:
+  version: 1.18.0
+
+ingress:
+  enabled: true
+  hosts:
+    - xxx
```

Comment:
What is that `xxx` for? Can we have a comment here, so people can understand what we're setting?

Reply:
If this is also just a default-value because the helm-charts validation CI needs it, then please remove it in favor of setting explicit values in the `ci/` sub directory of the `jaeger-operator` chart.

Reply:
these are the dummy values referenced in the default NOTES.txt...I have updated the NOTES, so that they are removed as well. (commit 7d10f69)